### PR TITLE
chore: add licenses badge

### PR DESCRIPTION
**Pull request by lukeed**
This service recursively checks/lists all licenses within a package's dependency graph. I remember there being a fair number of pieces involved here so was happy to see that all 400+ licenses involved are green!

Also, hello! 👋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 </p>
 <h1 align="center">
 	Microbundle
-	<a href="https://www.npmjs.org/package/microbundle"><img src="https://img.shields.io/npm/v/microbundle.svg?style=flat" alt="npm"></a> <a href="https://travis-ci.org/developit/microbundle"><img src="https://travis-ci.org/developit/microbundle.svg?branch=master" alt="travis"></a>
+	<a href="https://www.npmjs.org/package/microbundle"><img src="https://img.shields.io/npm/v/microbundle.svg?style=flat" alt="npm"></a> <a href="https://travis-ci.org/developit/microbundle"><img src="https://travis-ci.org/developit/microbundle.svg?branch=master" alt="travis"></a> <a href="https://licenses.dev/npm/microbundle"><img src="https://licenses.dev/b/npm/microbundle" alt="licenses" /></a>
```
Comment (lukeed) [inline README.md:6]: Wasn't sure what badge style to use, so will leave this here

```suggestion
	<a href="https://www.npmjs.org/package/microbundle"><img src="https://img.shields.io/npm/v/microbundle.svg?style=flat" alt="npm"></a> <a href="https://travis-ci.org/developit/microbundle"><img src="https://travis-ci.org/developit/microbundle.svg?branch=master" alt="travis"></a> <a href="https://licenses.dev/npm/microbundle"><img src="https://licenses.dev/b/npm/microbundle?style=flat" alt="licenses" /></a>
```

**Review by rschristian — APPROVED** [review]
Heyo 👋

File: README.md (hunk 1 above)
Comment (rschristian) [reply README.md:6]: Maybe my eyes are getting even worse but it looks like it matches pretty well already, despite the `?style=flat` on the NPM badge.

**Comment by rschristian** [comment]
Dunno what in the world caused compressed-size to fail, weird.

**Pull request merged**